bahmnicore-omod/src/main/java/org/bahmni/module/bahmnicore/web/v1_0/controller/VisitDocumentController.java

```diff
@@ -18,13 +19,17 @@
 import org.openmrs.module.webservices.rest.web.v1_0.controller.BaseRestController;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.HashMap;
 
 @Controller
 public class VisitDocumentController extends BaseRestController {
+    private static final String INVALID_USER_PRIVILEGE = "User [%d] does not have require to delete patient file [%s]";
+    private static final String DELETE_PATIENT_DOCUMENT_PRIVILEGE = "Delete Patient Document";
```

Comment:
Do you feel it would be a good idea that we create a tech debt card for pulling all "privilege" references into its own Enum/Constants file in bahmni-core -- so that one can see all defined privileges in one place, and people don't use "strings" in code. For instance in the TEST file (VisitDocumentControllerTest) - the string value has been used in this PR. If you agree, I will raise a card in JIRA for newbie volunteer contribution.

Reply:
sure, I was looking for a constants file - couldnt find any. Thought about introducing it , but there are many places we have to potentially do that. rather do it as tech debt - also, it will require some work in terms of how the beans have been registered and if we want to leverage the annotation based authorization. I suggest create a tech debt card - there are many things to be done. For the test, I will just declare the above as  public constant and use it from test case for now.

Reply:
Agree. It is a bigger change, which will need its own card. I will create one and add it to backlog. Thanks!